### PR TITLE
Added enable() and disable() methods for toggling ability to drag.

### PR DESCRIPTION
**Pull request by justindarc**

### CODE REVIEW

File: drag.js [hunk 1]
```diff
@@ -93,6 +94,37 @@ Drag.prototype.setupEvents = function() {
...
+    this.translate(this.startPosition.x, this.startPosition.y);
+
+    this.point = null;
+    this.startPosition = null;
+
+    clearTimeout(this.timeout);
+    this.timeout = setTimeout(() => this.dragging = false);
+
+    removeEventListener(pointer.move, this.onPointerMove);
+    removeEventListener(pointer.up, this.onPointerEnd);
```
Comment (aosmond) [inline drag.js]: Nit: Rather than duplicate the code, you could call onPointerEnd and not dispatch 'ended' if disabled?

Comment (justindarc) [inline drag.js]: Ah, good call. I was trying to figure out an elegant way to do that.

**Comment by aosmond** [comment]
r+

**Comment by justindarc** [comment]
Tests pass locally. Looks like an issue with Travis CI.

**Pull request merged**